gem/pupperware.gemspec

```diff
@@ -31,10 +31,4 @@ Gem::Specification.new do |spec|
   spec.bindir        = "exe"
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
-
-  spec.add_development_dependency "bundler", "~> 1.16"
-  spec.add_development_dependency "rake", "~> 10.0"
-
-  spec.add_runtime_dependency "rspec", "~> 3.0"
-  spec.add_runtime_dependency "json"
```

Comment:
Should probably set a minimum Ruby version here while we're at it...

Reply:
I think we have that already here: https://github.com/puppetlabs/pupperware/blob/master/gem/pupperware.gemspec#L16-L17